OrbitClientData/ProcessData.cpp

```diff
@@ -11,10 +11,11 @@
 #include "absl/strings/str_format.h"
 #include "process.pb.h"
 
-using orbit_client_protos::FunctionInfo;
 using orbit_grpc_protos::ModuleSymbols;
 using orbit_grpc_protos::ProcessInfo;
 
+ProcessData::ProcessData() { process_info_.set_pid(-1); }
```

Comment:
Do we have a constant somewhere for invalid pid instead of using -1?

Reply:
No, unfortunately not.